js-pkg/create-y-sweet-app/src/run.js

```diff
@@ -0,0 +1,108 @@
+import { exec } from 'node:child_process'
+import { cp, readdir } from 'node:fs/promises'
+import { dirname, resolve } from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { parseArgs } from 'node:util'
+import readline from 'node:readline'
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+})
+
+/**
+ * @param {string} prompt
+ * @returns {Promise<string>}
+ */
+function question(prompt, defaultValue = '') {
+  return new Promise((resolve) => rl.question(prompt, (result) => resolve(result || defaultValue)))
+}
+
+const { values, positionals } = parseArgs({ options: { template: { type: 'string', short: 't' } } })
+
+if (values.template) init(values.template, positionals[0])
+else help()
+
+function help() {
+  console.log('Usage: npm create y-sweet-app --template <template> [name]')
+}
+
+/**
+ * @param {string} template
+ * @param {string} project
+ */
+async function init(template, project) {
+  const TEMPLATES = new Set(['nextjs', 'remix'])
+
+  if (!TEMPLATES.has(template)) {
+    console.log(`No matching template for ${template}.`)
+    console.log('Usage: npm create y-sweet-app --template <template> [name]')
+    console.log('Available templates:', [...TEMPLATES].join(', '))
+    process.exit(1)
+  }
+
+  let name = project
+  if (!name)
+    name = await question(`What do you want to call your app? ${gray('(my-y-sweet-app)')} `)
+
+  let install = false
+  const installResponse = await question(
+    `Do you want to install dependencies with npm? ${gray('(Y/n)')} `,
+    'y',
+  )
+  if (installResponse.toLowerCase() === 'y') install = true
+
+  let git = false
+  const gitResponse = await question(
+    `Do you want to install dependencies with npm? ${gray('(Y/n)')} `,
+    'y',
+  )
+  if (gitResponse.toLowerCase() === 'y') git = true
```

Comment:
Worth adding the `git` and `install` options to the CLI options?

Reply:
I made this a wizard!

Reply:
I just meant as a way to skip those questions of the wizard. I think I've seen other wizards do this. Maybe also makes the command more reusable in automated environments? I don't think it's a blocker for release, though!